cmd/main.go

```diff
@@ -18,64 +18,144 @@ package main
 
 import (
 	"context"
+	"fmt"
+	"os"
+	"strings"
 	"time"
 
-	flag "github.com/spf13/pflag"
-
+	"github.com/kubernetes-sigs/aws-ebs-csi-driver/cmd/hooks"
+	"github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/cloud"
+	"github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/cloud/metadata"
 	"github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver"
 	"github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/metrics"
+	flag "github.com/spf13/pflag"
+	"k8s.io/component-base/featuregate"
 	logsapi "k8s.io/component-base/logs/api/v1"
 	json "k8s.io/component-base/logs/json"
-
 	"k8s.io/klog/v2"
 )
 
+var (
+	osExit      = os.Exit
+	featureGate = featuregate.NewFeatureGate()
```

Comment:
This one is helpful because the feature gate is used in two places:

1. `err := logsapi.AddFeatureGates(featureGate)`
2. `err = logsapi.ValidateAndApply(c, featureGate)`